pkgs/development/compilers/llvm/common/default.nix

```diff
@@ -507,9 +517,11 @@ let
       llvm = tools.libllvm;
 
       tblgen = callPackage ./tblgen.nix {
-        patches = builtins.filter
-          # Crude method to drop polly patches if present, they're not needed for tblgen.
-          (p: (!lib.hasInfix "-polly" p)) tools.libllvm.patches;
+        patches =
+          builtins.filter
+            # Crude method to drop polly patches if present, they're not needed for tblgen.
+            (p: (!lib.hasInfix "-polly" p))
+            tools.libllvm.patches;
```

Comment:
What changed here? Did it just get reformatted? If so, can that please be a separate commit so this is easier to review?

Reply:
Yes, it was caused by formatting.